### PR TITLE
Fixes #19 : [Feat] Implement parcelable interface for POJOs

### LINKED ISSUE #19: [Feat] Entities : add parcelable interface
SDK's entities don't implement parcelable interface, it would be nice to handle it

### PR DESCRIPTION
**Pull request by jcholin**
POJOs now implements Parcelable interface

### CODE REVIEW

File: sdk-api/src/main/java/com/xee/api/entity/Accelerometer.java [hunk 1]
```diff
@@ -31,6 +34,9 @@
     @SerializedName("date")
     private Date date;
 
+    public Accelerometer() {
+    }
```
Comment (quentin7b) [inline sdk-api/src/main/java/com/xee/api/entity/Accelerometer.java]: Maybe add default value for `nullables` (like date)

File: sdk-api/src/main/java/com/xee/api/entity/Car.java [hunk 2]
```diff
@@ -45,6 +47,9 @@
     @SerializedName("lastUpdateDate")
     private Date lastUpdateDate;
 
+    public Car() {
+    }
```
Comment (quentin7b) [inline sdk-api/src/main/java/com/xee/api/entity/Car.java]: Maybe add default value for `nullables` (like date)

File: sdk-api/src/main/java/com/xee/api/entity/Location.java [hunk 3]
```diff
@@ -35,6 +38,9 @@
     @SerializedName("date")
     private Date date;
 
+    public Location() {
+    }
```
Comment (quentin7b) [inline sdk-api/src/main/java/com/xee/api/entity/Location.java]: Maybe add default value for `nullables` (like date)

File: sdk-api/src/main/java/com/xee/api/entity/Signal.java [hunk 4]
```diff
@@ -68,6 +73,9 @@ public String toString() {
     @SerializedName("date")
     private Date date;
 
+    public Signal() {
+    }
```
Comment (quentin7b) [inline sdk-api/src/main/java/com/xee/api/entity/Signal.java]: Maybe add default value for `nullables` (like date)

File: sdk-api/src/main/java/com/xee/api/entity/Stat.java [hunk 5]
```diff
@@ -31,6 +36,9 @@
     @SerializedName("value")
     private T value;
 
+    public Stat() {
+    }
```
Comment (quentin7b) [inline sdk-api/src/main/java/com/xee/api/entity/Stat.java]: Maybe add default value for `nullables` (like beginDate or endDate)

**Review by qk7b — APPROVED** [review]

**Comment by jcholin** [comment]
It closes #19

**Pull request merged**